src/screens/FeedScreen/Content.js

```diff
@@ -318,7 +318,12 @@ const Content = ({
             fontSize={normalizeFontSizeByWidth(14)}
             text={message}
             topicContainer={{
-              position: item.post_type === POST_TYPE_STANDARD ? 'absolute' : 'relative',
+              position:
+                item.post_type === POST_TYPE_POLL
+                  ? item.pollOptions.length > 3
+                    ? 'relative'
+                    : 'absolute'
+                  : 'absolute',
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The nested ternary operators here make the code hard to read and understand. It's generally a good practice to avoid nested ternary operators for the sake of readability and maintainability. Consider refactoring this into if/else statements or a separate function.

```diff
-              position:
-                item.post_type === POST_TYPE_POLL
-                  ? item.pollOptions.length > 3
-                    ? 'relative'
-                    : 'absolute'
-                  : 'absolute',
+              position: getPositionBasedOnPostType(item),
```

And then define the `getPositionBasedOnPostType` function like so:

```javascript
function getPositionBasedOnPostType(item) {
  if (item.post_type === POST_TYPE_POLL) {
    return item.pollOptions.length > 3 ? 'relative' : 'absolute';
  }
  return 'absolute';
}
```
This way, the logic is more clear and the code is easier to maintain.


<!-- This is an auto-generated comment by OSS CodeRabbit -->